### PR TITLE
Remove event filtering

### PR DESCRIPTION
**Pull request by csillag**
As discussed on the task refinement meeting

### CODE REVIEW

File: .changelog/1174.trivial.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Simplify UI: remove event filtering switch
```
Comment (lukaw3d) [inline .changelog/1174.trivial.md:1]: This is more `breaking` than `trivial`

File: src/app/pages/BlockDetailPage/EventsCard.tsx [hunk 2]
```diff
@@ -34,11 +29,7 @@ const EventsList: FC<{ scope: SearchScope; blockHeight: number; filterMode: Even
 
   const { isLoading, isError, data } = eventsQuery
 
-  const events = data?.data.events.filter(
-    event =>
-      !event.tx_hash && // TODO: remove filtering here if it's implemented using the query parameters
-      (filterMode === EventFilterMode.All || event.type !== RuntimeEventType.accountstransfer),
-  )
+  const events = data?.data.events.filter(event => !event.tx_hash)
```
Comment (lukaw3d) [inline src/app/pages/BlockDetailPage/EventsCard.tsx:32]: We want to show all events

Comment (csillag) [reply src/app/pages/BlockDetailPage/EventsCard.tsx:32]: This is on the block page. Removing this filter will also includes events that belong to a tx. Is that what we want?

Comment (lukaw3d) [reply src/app/pages/BlockDetailPage/EventsCard.tsx:32]: yes

File: .changelog/1174.trivial.md (hunk 1 above)
Comment (csillag) [reply .changelog/1174.trivial.md]: OK renamed accordingly

File: src/app/pages/BlockDetailPage/EventsCard.tsx (hunk 2 above)
Comment (csillag) [reply src/app/pages/BlockDetailPage/EventsCard.tsx]: OK, removed all filtering from there.

**Review by lukaw3d — APPROVED** [review]

**Pull request merged**